planner/core/logical_plan_builder.go

```diff
@@ -4470,6 +4471,14 @@ func (b *PlanBuilder) handleDefaultFrame(spec *ast.WindowSpec, windowFuncName st
 		}
 		return &newSpec, true
 	}
+	if needFrame && spec.Frame != nil && spec.OrderBy == nil {
```

Comment:
`spec.OrderBy` should not be the problem using `aggWindowProcessor`, you can judge this here (https://github.com/pingcap/tidb/blob/ce849c3e19b0b69404848b0eee05356dcd81277b/executor/builder.go#L3323)